docs/src/Color.doc.js

```diff
@@ -0,0 +1,61 @@
+// @flow strict
+import type { Node } from 'react';
+import { Flex } from 'gestalt';
+import MainSection from './components/MainSection.js';
+import PageHeader from './components/PageHeader.js';
+import ColorPalette from './components/ColorPalette.js';
+
+const cards: Array<Node> = [];
+const card = (c) => cards.push(c);
+
+card(
+  <PageHeader
+    name="Color"
+    description={`
+    Our color palettes are shared between Brand and Gestalt, and represent our full range of options. For further Brand guidance, please reference the [Brand guidelines website](https://brand.pinterest.com/).
```

Comment:
Do we have any other design guidance we should provide here, or are we deferring to the Brand guidelines for that? Should we establish what our primary/secondary/tertiary colors are, call out Shopping blue, anything like that? Should there be any mention of dark mode here?

Reply:
This is just the initial representation of the color palette, so next step is to create a Design Tokens page that showcases our Alias tokens, which is when dark mode and primary/secondary/etc. would come in